test/components/views/messages/TextualBody-test.tsx

```diff
@@ -187,10 +187,10 @@ describe("<TextualBody />", () => {
                 event: true,
             });
 
-            const wrapper = getComponent({ mxEvent: ev }, matrixClient);
-            expect(wrapper.text()).toBe("foo baz bar del u");
-            const content = wrapper.find(".mx_EventTile_body");
-            expect(content.html()).toBe(
+            const { container } = getComponent({ mxEvent: ev }, matrixClient);
+            expect(container).toHaveTextContent("foo baz bar del u");
+            const content = container.querySelector(".mx_EventTile_body");
```

Comment:
I think testing against a snapshot fits better than querying class names and checking for HTML.

Applies to other places too.